guide/popular-topics/threads.md

```diff
@@ -0,0 +1,157 @@
+# Threads
+
+Threads can be thought of as temporary sub-channels inside an existing channel, to help better organize conversation in a busy channel.
+
+## Thread related gateway events
+
+::: tip
+You can use the `isThread()` typeguard to make sure the channel you're working with is a ThreadChannel!
+:::
+
+Threads introduce a number of new gateway events, which are listed below:
+
+- `Client#threadCreate`: Emitted whenever a thread is created or when the client user is added to a thread.
+- `Client#threadDelete`: Emitted whenever a thread is deleted.
+- `Client#threadUpdate`: Emitted whenever a thread is updated - e.g. name change, archive state change, locked state change.
+- `Client#threadListSync`: Emitted whenever the client user gains access to a text or news channel that contains threads.
+- `Client#threadMembersUpdate`: Emitted whenever members are added or removed from a thread. Requires `GUILD_MEMBERS` privileged intent.
+- `Client#threadMemberUpdate`: Emitted whenever the client user's thread member is updated.
+
+## Creating and deleting threads
+
+Threads are created and deleted using the `ThreadManager` of a text or news channel.
+To create a thread you call the `ThreadManager#create()` method:
+
+```js {2-8}
+client.on('messageCreate', async message => {
+	const thread = await message.channel.threads
+		.create({
+			name: 'food-talk',
+			autoArchiveDuration: 60,
+			reason: 'Needed a separate thread for food',
+		});
+	console.log(`Created thread: ${thread.name}`);
+});
```

Comment:
```suggestion
Threads are created and deleted using the `ThreadManager` of a text or news channel.
To create a thread you call the `ThreadManager#create()` method:

<!-- eslint-skip -->

```js
const thread = await message.channel.threads.create({
	name: 'food-talk',
	autoArchiveDuration: 60,
	reason: 'Needed a separate thread for food',
});

console.log(`Created thread: ${thread.name}`);
```

Reply:
Removing the `client.on('messageCreate', ...)`  wrapper since it doesn't entirely make sense. Unfortunately it means adding an `<!-- eslint-skip -->` comment at each codeblock because of top level await usage, but they're all small snippets so it's no big deal.